packages/core/src/kilt/Kilt.ts

```diff
@@ -50,4 +40,31 @@ export async function init<K extends Partial<ConfigService.configOpts>>(
   config(configs || {})
   await cryptoWaitReady()
 }
-export const { disconnect } = BlockchainApiConnection
+
+/**
+ * Connects to the KILT Blockchain using the api instance set with `init()`.
+ *
+ * @param wsEndpoint
```

Comment:
I think we were calling this `address` before, should we keep the name?

Reply:
mhhhh, maybe... it was a pretty bad name though given that we also have cryptographic addresses

Reply:
Good point. We also have DID service endpoints, however. 

I think we can have `Uri` in the name because it is always a URI. And it kind of defines the blockchain you connect to, so `blockchainUri` would make sense to me. A `blockchainNodeUri` if you’d like. I would avoid using "full node" here though, as these words are not really clear to an external developer. (Could there also be a way to connect to a not-full node?).

Which also leads me to think that maybe we should then align the name of `FullNode…Error` with that.

Reply:
It is also always a URL, isn't it? And more specifically, it's the URL where a Blockchain node exposes an RPC endpoint. Is JSON-RPC a concept people know of? Then we could call it `blockchainRpcUrl` or something. Or `blockchainRpcWsUri` to make it clearer that we expect a ws/wss url here

Reply:
Yeah, I guess `Url` is more fitting than `Uri` here :shrug: 

The RPC is definitely known, I like `blockchainRpcWsUrl`.

Reply:
Will `FullNodeConnectionNotSetError` then become `BlockchainRpcConnectionNotSetError`? Actually I avoid using "not" in names, so maybe even `BlockchainApiMissingError`?

Reply:
sounds good